pkgs/development/libraries/science/math/cudnn/generic.nix

```diff
@@ -1,8 +1,14 @@
 { stdenv
 , lib
-, cudatoolkit
+, zlib
+, useCudatoolkitRunfile ? false
+, cudaVersion
+, cudaMajorVersion
+, cudatoolkit # if cuda>=11: only used for .cc
+, libcublas ? null # cuda <11 doesn't ship redist packages
```

Comment:
```suggestion
, libcublas # cuda <11 doesn't ship redist packages
```

Reply:
This must be `? null` because for `cudaPacakges_{10,...,11_3}` there's no `libcublas`